scripts/gcs-inventory-loader.ipynb

```diff
@@ -0,0 +1,490 @@
+    "\n",
+    "fig, ax = plt.subplots()\n",
+    "patches, text = plt.pie(sizes, radius=5, startangle=90)\n",
+    "\n",
+    "\n",
+    "plt.legend(patches, labels, loc=\"right\", bbox_to_anchor=(-0.1, 1.),\n",
+    "           fontsize=10)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Duplicate Files\n",
+    "\n",
+    "Using the file path and Hash number combined, the queries finds all the duplicate files and returns a list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#If you didn't run it before\n",
+    "%load_ext google.cloud.bigquery"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bigquery file_Duplicate --use_rest_api --params $params\n",
+    "DECLARE path STRING;\n",
+    "SET path = @path;\n",
+    "EXECUTE IMMEDIATE CONCAT('SELECT Concat(regexp_replace(regexp_replace(name, cast(ARRAY_REVERSE(SPLIT(name, r\"/\"))[SAFE_OFFSET(0)] as string), r\"\"), cast(ARRAY_REVERSE(SPLIT(name, r\"/\"))[SAFE_OFFSET(1)] as string), r\"\"), r\"-\", cast( md5Hash as string)) as file_name, Count(regexp_replace(regexp_replace(name, cast(ARRAY_REVERSE(SPLIT(name, r\"/\"))[OFFSET(0)] as string), r\"\"), cast(ARRAY_REVERSE(SPLIT(name, r\"/\"))[SAFE_OFFSET(1)] as string), r\"\")) as count FROM `',path,'` Group by file_name Having Count(file_name) > 2 ORDER By COUNT DESC')                     "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "pd.set_option(\"display.max_rows\", None, \"display.max_columns\", None, 'display.max_colwidth', None)\n",
+    "file_Duplicate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Timeline\n",
+    "\n",
+    "This Bar graph shows a timeline of when files were created"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#If you didn't run it before\n",
+    "%load_ext google.cloud.bigquery"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bigquery data_timeline_table --use_rest_api --params $params\n",
+    "DECLARE path STRING;\n",
+    "SET path = @path;\n",
+    "EXECUTE IMMEDIATE CONCAT('SELECT DATE(timeCreated) as date,  count(name) as name_count FROM `',path,'` Group By DATE(timeCreated)');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_timeline_table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
```

Comment:
you only need to run the `%matplotlib inline` command once - you've already run it up above, no need to re-run here (can delete this cell)